java/src/main/kotlin/io/spine/validation/java/JavaValidationPlugin.kt

```diff
@@ -26,48 +26,21 @@
 
 package io.spine.validation.java
 
-import io.spine.protodata.plugin.Plugin
-import io.spine.protodata.render.Renderer
-import io.spine.server.BoundedContextBuilder
 import io.spine.validation.ValidationPlugin
-import io.spine.validation.java.point.PrintValidationInsertionPoints
-import io.spine.validation.java.setonce.SetOnceValidationRenderer
+import io.spine.validation.java.setonce.SetOnceRenderer
 
 /**
- * A plugin that sets up everything needed to generate Java validation code.
+ * An implementation of [ValidationPlugin] for Java language.
  *
- * This plugin uses a delegate plugin to set up some of the components needed for
- * code generation. By default, a [ValidationPlugin] is used. However, API users may
- * extend this plugin's behavior and supply a more rich base plugin.
- *
- * @param base The base plugin to extend.
- * @constructor Creates an instance that contains all the components from the given [base] plugin.
+ * Note: [`(set_once)`][SetOnceRenderer] option is rendered with its own
```

Comment:
It's not the option which is rendered. We render validation code serving the `(set_once)` constraint.

I would start the explanation of the matter with the enumeration of the renderers included into the `ValidationPlugin`, and then explain them both briefly.